PowerShell-REST/RestHowTo-Invoke-RestMethod.ps1

```diff
@@ -0,0 +1,189 @@
+# Admin api-key and service name can be found in the portal
+# Given https://mydemo.search.windows.net, service name is "mydemo"
+$apiKey = '[Put your admin api-key here]'
+$searchServiceName = '[Put your search service name here]'
+
+function Write-Line
+{
+    param(
+        [Parameter(Mandatory=$True,ValueFromPipeline=$True)]$text
+    )
+    Write-Host $text -foregroundColor Green
+}
+
+function Write-Headers
+{
+    param(
+        [Parameter(Mandatory=$True,ValueFromPipeline=$True)]$headers
+    )
+
+    $text = ($headers.Keys | foreach { "$_`: $($headers[$_])" }) -join "`n"
+    Write-Line $text
+}
+
+function Execute-Request
+{
+    param($method, $requestUri, $apiKey)
+    # Adding 'odata.metadata=none' to the Accept header to make the response payloads more concise and readable.
+    $headers = @{"api-key"=$apiKey; "Accept"="application/json; odata.metadata=none"}
+
+    Write-Line "`n$method $requestUri"
+    Write-Headers $headers
+
+    $resp = @{}
+    $resp = Invoke-WebRequest $requestUri -Method $method -Headers $headers
+    $resp.StatusCode
+    $resp.Content
+}
+
+function Execute-RequestWithBody
+{
+    param($method, $requestUri, $apiKey, $body)
+    # Adding 'odata.metadata=none' to the Accept header to make the response payloads more concise and readable.
+    $headers = @{"api-key"=$apiKey; "Accept"="application/json; odata.metadata=none"}
+
+    # Adding 'charset=utf-8' to the Content-Type header so that we can index non-ASCII characters like accents from PowerShell.
+    $contentType = "application/json; charset=utf-8"
+
+    Write-Line "`n$method $requestUri"
+    Write-Line "Content-Type: $contentType"
+    Write-Headers $headers
+    Write-Line "`n$body"
+    
+    $resp = @{}
+    $resp = Invoke-RestMethod $requestUri -Method $method -Body $body -ContentType $contentType -Headers $headers
+
+    $resp.StatusCode
+    $resp.Content
+}
+
+$body = @"
```

Comment:
Is this from the `search-rest-api-getting-started` repo? Will you send a PR to remove the content from there and put a redirection message w/ link in that README?